charts/gke.md

```diff
@@ -0,0 +1,87 @@
+# Install Zenko On Google Cloud Kubernetes Engine
+
+Zenko can be installed on a cluster managed on Google Cloud (GKE).
+Start a new cluster on Kubernetes following the instructions on [Google
+Cloud
+documentation](https://cloud.google.com/kubernetes-engine/docs/quickstart).
+
+To run Zenko you'll need a cluster with 3 nodes, 2vCPU and 7.5GB RAM
+each. Once the cluster is running, connect to it and install Helm on
+it.
+
+## Create Role For Tiller
+
+Google Kubernetes Engine requires Role Based Access Control to be
+setup. The first step is to create a `serviceaccount` for `tiller`:
+
+```shell
+$ kubectl create serviceaccount tiller --namespace kube-system
+```
+
+## Install Helm on GKE Cluster
+
+Strangely, Helm is not available by default on GKE and needs to be
+installed.
+
+```shell
+$ curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get > get_helm.sh
+$ bash ./get_helm.sh
+```
+
+Once that's completed, start Helm:
+
+```shell
+$ helm init --service-account tiller --wait
```

Comment:
Make sure the right kubecontext is set up etc. first, e.g. by asking the user to run `kubectl get nodes` before and validate the output.